js/interface.js

```diff
@@ -25,6 +25,7 @@ function save(notifyComplete) {
   Fliplet.Widget.save(data).then(function () {
     if (notifyComplete) {
       Fliplet.Widget.complete();
+      Fliplet.Studio.emit('reload-page-preview');
```

Comment:
@hcarneiro it's necessary to reload the page preview because the image html does not have the src anymore.